src/tokens/theme.web.js

```diff
@@ -97,25 +118,44 @@ const darkTheme = {
       600: colors.azzure[600],
       800: colors.azzure[800],
       900: colors.azzure[900],
-    },
-    secondary: {
-      600: colors.lilac[600],
-      900: colors.lilac[900],
+      920: colors.azzure[920],
+      930: colors.azzure[930],
+      950: colors.azzure[950],
+      970: colors.azzure[970],
     },
     negative: {
-      300: colors.rose[300],
-      600: colors.rose[600],
       900: colors.rose[900],
+      930: colors.rose[930],
+      960: colors.rose[960],
     },
     positive: {
-      300: colors.green[300],
-      600: colors.green[600],
       900: colors.green[900],
+      930: colors.green[930],
+      960: colors.green[960],
     },
     neutral: {
-      300: colors.honey[300],
-      600: colors.honey[600],
       900: colors.honey[900],
+      930: colors.honey[930],
+      960: colors.honey[960],
+    },
+    exceptions: {
```

Comment:
It will break. We don't have this structure. All you need to do is have this as top level objects don't nest it under `exception`